File: dayFour/javaScript/After Break/TaskEight.js
const prompt = require("prompt-sync")();
let number = 0;
let sum = 0;
let count = 1;
while(count <= 10){
    let number = Number(prompt("Enter a number: "));
    if(number < 0 || number > 100){
        console.log("Invalid input(Enter number between 1 to 100)");
    }
    else {
    sum += number;
    count++;
    }
}
console.log("sum is: " +  sum);

File: dayFour/python/After Break/task_eight.py
total = 0
count = 1
while count <= 10 :
    number = int(input("Enter a number: "))
    if number < 0 or number > 100 :
        print("Invalid input(Enter number between 1 to 100)")
    else :
        total += number;
        count += 1

print("sum is: ", total)


